framework/src/utils/PerfGraph.C

```diff
@@ -237,13 +357,14 @@ PerfGraph::recursivelyPrintGraph(PerfNode * current_node,
                                  unsigned int level,
                                  unsigned int current_depth)
 {
-  mooseAssert(_id_to_section_name.find(current_node->id()) != _id_to_section_name.end(),
+  mooseAssert(_id_to_section_info.find(current_node->id()) != _id_to_section_info.end(),
               "Unable to find section name!");
 
-  auto & name = current_node->id() == 0 ? _root_name : _id_to_section_name[current_node->id()];
+  auto & name = _id_to_section_info[current_node->id()]._name;
 
-  mooseAssert(_id_to_level.find(current_node->id()) != _id_to_level.end(), "Unable to find level!");
-  auto & node_level = _id_to_level[current_node->id()];
+  mooseAssert(_id_to_section_info.find(current_node->id()) != _id_to_section_info.end(),
+              "Unable to find level!");
+  auto & node_level = _id_to_section_info[current_node->id()]._level;
```

Comment:
This can all be replaced with:

```
mooseAssert(_id_to_section_info.count(current_node->id()), "Could not find");
const auto & entry = _id_to_section_info[current_node->id()];
auto & name = entry._name;
auto & node_level = entry._level;
```